qiita_db/metadata_template/test/test_prep_template.py

```diff
@@ -378,19 +377,14 @@ def tearDown(self):
         for f in self._clean_up_files:
             remove(f)
 
-
-class TestPrepTemplateReadOnly(BaseTestPrepTemplate):
-    def setUp(self):
-        self._set_up()
-
     def test_study_id(self):
         """Ensure that the correct study ID is returned"""
         self.assertEqual(self.tester.study_id, 1)
 
     def test_init_unknown_error(self):
         """Init raises an error if the id is not known"""
         with self.assertRaises(qdb.exceptions.QiitaDBUnknownIDError):
-            qdb.metadata_template.prep_template.PrepTemplate(3)
+            qdb.metadata_template.prep_template.PrepTemplate(30000)
```

Comment:
Correct - since we are not reseting the database and we are creating a lot of templates for testing, the pt id with id 3 ends up existing, and hence making this test to fail. By changing it to a really big number we avoid that collision.